src/main/resources/messages.properties

```diff
@@ -429,6 +429,7 @@ upload-documents.this-file-is-too-large=This file is too large and cannot be upl
 upload-documents.this-pdf-is-password-protected=This PDF is password protected. Try removing the password or uploading a screenshot instead.
 upload-documents.this-pdf-is-in-an-old-format=This PDF is in an old format. Try converting it to an image or uploading a screenshot instead.
 upload-documents.there-was-an-issue-on-our-end=There was an issue processing this file on our end. Sorry about that! Please try another file or upload a screenshot instead.
+upload-documents.there-is-a-problem-with-the-image=This image cannot but uploaded to your application.
```

Comment:
Typo here. Might be worth asking design/product what they'd like this to say if you haven't asked already

Reply:
Hey I talked to design and I believe we have the right wording.  I'm going to mark this as resolved